recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ about:
   home: https://github.com/rtfd/recommonmark
   license: MIT
   license_family: MIT
-#  license_file: LICENSE.md
+  # license_file: license.md # Their license.md is not distributed in their pypi package.
```

Comment:
Can you make an issue on this at the source?

Reply:
I think it is fixed in their master: https://github.com/rtfd/recommonmark/blob/master/MANIFEST.in#L1

Reply:
Brilliant!